Documentation/Plugins/Index.rst

```diff
@@ -1084,3 +1084,76 @@ Plugin allows also to configure (searchHlParameters) by which URL parameters wor
 The fulltext is fetched and rendered by JavaScript into the `<div id="tx-dlf-fulltextselection">` of the pageview plugin.
 
 **Please note**: To allow JavaScript fetching the fulltext, the `CORS headers <https://en.wikipedia.org/wiki/Cross-origin_resource_sharing>`_ must be configured appropriate on the providing webserver.
+
+
+Search in Document Tool
+^^^^^^^^^^^^^
```

Comment:
```suggestion
Search in Document Tool
^^^^^^^^^^^^^^^^^^^^^^^
```
The underline of headings must be as long as the heading itself. See e.g. https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#headings